commit edcbe5f52ddfceee3d66d69bbcebbceac06b6d0d
Author: Adam Jackson <ajax@redhat.com>
Date:   Tue Feb 4 16:38:06 2020 -0500

    Fix link failure with gcc 10
    
    Without the 'extern' this looks like a definition not just a
    declaration, in every file that includes the header. gcc 10 is stricter
    about this kind of multiple definition.

diff --git a/src/drmmode_display.h b/src/drmmode_display.h
index 803ac3c..9c0f25a 100644
--- a/src/drmmode_display.h
+++ b/src/drmmode_display.h
@@ -289,7 +289,7 @@ Bool drmmode_wait_vblank(xf86CrtcPtr crtc, drmVBlankSeqType type,
 			 uint64_t *ust, uint32_t *result_seq);
 
 
-miPointerSpriteFuncRec drmmode_sprite_funcs;
+extern miPointerSpriteFuncRec drmmode_sprite_funcs;
 
 
 #endif
